Dockerfile-hydroshare

```diff
@@ -3,7 +3,7 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
-RUN pip install markdown
```

Comment:
@cuahsimarko markdown is already moved into hs_docker_base image, so this pip install markdown line should be removed.

Reply:
Fixed in 8df0ec4e6